ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -709,6 +731,7 @@ let start () =
   let on_request rpc state caps req =
     prepare_and_run @@ fun () -> on_request rpc state caps req
   in
+  log ~title:"info" "starting";
```

Comment:
This logging doesn't seem very helpful

Reply:
Ah I added it for testing and forgot to remove